shell/initialize/entry-helpers.js

```diff
@@ -54,6 +54,16 @@ export const loadDebugger = (vueApp) => {
   }
 };
 
+/**
+ * Handle errors with a redirect
+ * @param {*} context 
+ * @param {*} message 
+ */
+const errorRedirect = (context, message) => {
```

Comment:
I have noticed that we are using something else for errors which is a leftover of Nuxt, instead of the `fail-whale` path.
For some reason a simple action commit is not sufficient, so I've set a function wrapper and forced redirect after setting the error.